src/types/pill-category.ts

```diff
@@ -3,4 +3,6 @@ export type PillCategory =
     | 'Demo App'
     | 'Tutorial'
     | 'Podcast'
-    | 'Video';
+    | 'Video'
+    | 'Quickstart'
+    | 'Code Example';
```

Comment:
Not necessary, but should we think about renaming this from `PillCategory` to something more informative? Like `ContentType`? I think it could get confusing for someone reading this in the future.